packages/vhd-lib/createStreamNbd.js

```diff
@@ -14,6 +14,9 @@ const {
 const { fuHeader, checksumStruct } = require('./_structs')
 const assert = require('node:assert')
 
+const MAX_DURATION_BETWEEN_PROGRESS_EMIT = 5e4
```

Comment:
```suggestion
const MAX_DURATION_BETWEEN_PROGRESS_EMIT = 5e3
```
I think 50 second is a little long between update , the right value should be between 5 and 30 seconds